src/lldp_syncd/daemon.py

```diff
@@ -44,12 +44,17 @@ def parse_time(time_str):
     }
     :return: parsed age in time ticks (or seconds)
     """
-    days, hour_min_secs = re.split(LLDPD_UPTIME_RE_SPLIT_PATTERN, time_str)
-    struct_time = time.strptime(hour_min_secs, LLDPD_TIME_FORMAT)
-    time_delta = datetime.timedelta(days=int(days), hours=struct_time.tm_hour,
-                                    minutes=struct_time.tm_min,
-                                    seconds=struct_time.tm_sec)
-    return int(time_delta.total_seconds())
+    secs = 0
+    try:
+        days, hour_min_secs = re.split(LLDPD_UPTIME_RE_SPLIT_PATTERN, time_str)
+        struct_time = time.strptime(hour_min_secs, LLDPD_TIME_FORMAT)
+        time_delta = datetime.timedelta(days=int(days), hours=struct_time.tm_hour,
+                                        minutes=struct_time.tm_min,
+                                        seconds=struct_time.tm_sec)
+        secs = int(time_delta.total_seconds())
```

Comment:
[minor] you don't need the `secs` variable here. It is not used outside of the try block so you can return the value as before: `return int(time_delta.total_seconds())`. This will make the code more readable.

Reply:
Handled